controllers/controllers.js

```diff
@@ -16,4 +14,21 @@ exports.getEndpointsInfo = (req, res, next) => {
        res.status(200).send({endpoints: JSON.parse(result)})
     })
     .catch((err) => {next(err)})
+}
+
+exports.getArticleById = (req, res, next) => {
+    const {article_id} = req.params;
+    fetchArticleById(article_id).then((result) => {
```

Comment:
We could have handled this in the model and rejected the promise in order to pipe this error through our error handling middleware